src/neptune/common/warnings.py

```diff
@@ -68,6 +87,7 @@ def warn_once(message: str, *, exception: type(Exception) = None):
         message_hash = hash(message)
 
         if message_hash not in warned_once:
+            warnings.formatwarning = format_message
```

Comment:
I think that I have to restore the old method after the message is posted. It's probably applied globally even outside of Neptune package.

Reply:
Fixed